packages/tools/src/BoltTool.ts

```diff
@@ -0,0 +1,125 @@
+import path from "path";
```

Comment:
I think that perhaps it's time to drop this tool - the project has stalled, isn't actively maintained and has a minimal amount of downloads